src/PowerShellEditorServices/Hosting/EditorServicesHost.cs

```diff
@@ -291,6 +292,25 @@ public void StartLogging(string logFilePath, PsesLogLevel logLevel)
         {
             _logger.LogInformation($"Debug NamedPipe: {config.InOutPipeName}\nDebug OutPipe: {config.OutPipeName}");
 
+            IServiceProvider serviceProvider = null;
+            if (!useExistingSession)
```

Comment:
Is there a reason for not using `useTempSession` with inverted logic? That would seem to line up with the conceptual feature better to me, and feels like it aligns with "breaking out" of the normal code when the `useTempSession` switch is set.

Reply:
Yeah that's fine - `useExistingSession` was just "what was there before"

Reply:
changed it to `useTempSession` hopefully that's a bit more clear.